hack/update-deps.sh

```diff
@@ -29,3 +29,8 @@ go mod vendor
 # Delete all vendored broken symlinks.
 # From https://stackoverflow.com/questions/22097130/delete-all-broken-symbolic-links-with-a-line
 find vendor/ -type l -exec sh -c 'for x; do [ -e "$x" ] || rm "$x"; done' _ {} +
+
+
+# HACK HACK HACK:
+# Until we can cleanly remove it, copy root main.go and move it to ./cmd/ko
+cp ${PROJECT_ROOT}/main.go ${PROJECT_ROOT}/cmd/ko/
```

Comment:
@ImJasonH ^

Is it possible to just commit a symlink to the root? Or would that break everything?

Reply:
Missed this 😇

I'd prefer something explicit since it also gives us an opportunity to inject a deprecation warning

Reply:
Added a message, dropped the copy,